pkg/server/app/app.go

```diff
@@ -328,6 +330,33 @@ func checkForProtectedEnvVars(evsNames []string) error {
 	return nil
 }
 
+func (ops *AppOperations) List(user *storage.User) ([]*List, error) {
+	teams, err := ops.tops.ListByUser(user.Email)
+	if err != nil {
+		return nil, err
+	}
+	lists := make([]*List, 0)
+	for _, team := range teams {
+		appNames, err := ops.kops.FindAppByLabel(team.Name)
+		if err != nil {
+			return nil, err
+		}
+		for _, app := range appNames {
+			appAdd, err := ops.kops.AddressList(string(app))
```

Comment:
Sincerely, I do not remember why I needed to convert the `app` value to string. I think that I used to convert rune to string. But I just tested it and it worked fine. Removed it. Tks